site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -810,8 +810,10 @@ components:
          properties:
             "type":
                type: string
+               pattern: [a-z0-9-]
```

Comment:
We should update a default patterns for different identifiers and add some docs on the configuration.

Reply:
done :)